unit_test/test.cpp

```diff
@@ -103,8 +107,36 @@ TEST(PFVariables, saveload) {
 }
 
 // Similar to above, the python implementation was modified to use the periodic initial conditions
-TEST(PFHub1aPeriodic, periodic) {
-    PFHub1aPeriodic simulation(96, .5);
+TEST(PFHub1aCHiMaD2023, FullRun) {
+    PFHub1aCHiMaD2023 simulation(96, .5);
+    simulation.initialize();
+    auto results = simulation.get_cpu_view();
+    EXPECT_NEAR(0.53, results(0, 0, 0), 1e-9);
```

Comment:
Let's do this in a helper function (with the view as a template input arg) rather than copying